service/src/main/java/org/broadinstitute/listener/relay/http/TargetHttpResponse.java

```diff
@@ -81,8 +86,15 @@ public static TargetHttpResponse createTargetHttpResponse(
           .forEach(
               (key, value) -> {
                 String headerValue = value.iterator().next();
-                responseHeaders.put(key, headerValue);
+                if (headerValue != ACCESS_CONTROL_ALLOW_ORIGIN && headerValue != CONTENT_SECURITY_POLICY)
+                  responseHeaders.put(key, headerValue);
               });
+
+      responseHeaders.put(ACCESS_CONTROL_ALLOW_ORIGIN, "*");
+      responseHeaders.put(ACCESS_CONTROL_ALLOW_CREDENTIALS, "true");
+      responseHeaders.put(ACCESS_CONTROL_ALLOW_HEADERS, "Authorization, Content-Type, Accept, Origin, X-App-Id");
+      responseHeaders.put(ACCESS_CONTROL_MAX_AGE, "1728000");
```

Comment:
I believe we will need
`Access-Control-Allow-Methods` with `OPTIONS, GET,`. Leo sets all `OPTIONS, POST, PUT, GET, DELETE, HEAD, PATCH`, so might be safer to include all.

Not sure if we need to set the `X-Frame-Options` header as well...

Reply:
yea, that's for preflight....which I didn't know where that'll be..but just figured it out with Jesus